### ISSUE TITLE
Potentially: return known keywords (symbols?)

### ISSUE THREAD

**Issue by benjamin-asdf** [babashka/babashka.nrepl#55]
with bb nrepl. 
Try to complete some keyword that you know sci already evalved. 
In jvm cider it gives you all keywords loaded. This behaviour is enjoyable in my opintion. 
It allows me to for example eval aws/ops once and then have complete for all the keys.

**Comment by borkdude** [comment]
Sounds good. @ericdallo: does clojure-lsp do a similar thing?

**Comment by ericdallo** [comment]
Yep, clojure-lsp completion return all known keywords on local buffer and if namespaced/aliased bring it corresponding namespace keywords as well

**Comment by borkdude** [comment]
Do you mean, if namespaced, then global completions instead of only local buffer?

**Comment by ericdallo** [comment]
Yes

**Comment by benjamin-asdf** [comment]
Also `:require` and such. I'm guessing this will be covered automatically but mentioning it.

**Comment by benjamin-asdf** [comment]
@borkdude do you have an idea how to go about implementing this?

**Comment by borkdude** [comment]
> In jvm cider it gives you all keywords loaded

How does JVM cider do this?

**Comment by benjamin-asdf** [comment]
Turns out is depends on https://github.com/alexander-yakushev/compliment it has a namespace for keywords.
```clojure 
import java.lang.reflect.Field

(def keywords-table
  (memoize
   (fn []
     (let [^Field field (.getDeclaredField clojure.lang.Keyword "table")]
       (.setAccessible field true)
       (.get field nil)))))

```
